### PR TITLE
3601_Handler_Event

### PR DESCRIPTION
**Pull request by dinhquang251**
https://edu-redmine.framgia.vn/issues/3601

### CODE REVIEW

**Comment by dinhquang251** [comment]
Handler Load data to RecyclerView
https://edu-redmine.framgia.vn/issues/3601

**Comment by dinhquang251** [comment]
ready

File: app/src/main/java/com/framgia/quangtran/music_42/ui/personal/PersonalFragment.java [hunk 1]
```diff
@@ -1,26 +1,44 @@
...
+    private static final int READ_EXTERNAL_STORAGE = 1;
+    private boolean isPermission = false;
     private LinearLayout mLinearLocal;
     private LinearLayout mLinearDownload;
     private LinearLayout mLinearFavorite;
-
+    private RecyclerView mRecyclerRecent;
+    private PersonalContract.Presenter mPresenter;
+    private List<Track> mTracks;
+    TodayAdapter mTodayAdapter;
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/quangtran/music_42/ui/personal/PersonalFragment.java]: access modifier

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by dinhquang251** [comment]
ready

**Pull request merged**